assets/javascripts/test_result.js

```diff
@@ -745,40 +745,56 @@ function renderInvestigationTab(response) {
 
     var tbodyElement = document.createElement('tbody');
     Object.keys(response).forEach(key => {
+        var spl = key.split(':');
+        var value = response[key];
+        var type = "pre";
+
```

Comment:
How about documenting the format explicitly?

```suggestion

        // Parse optional type specifier, e.g. "last_good:link"
```